.travis.yml

```diff
@@ -28,5 +90,13 @@ script:
   - make
 
 after_success:
-  - make check
-  - if type cppcheck &> /dev/null ; then cppcheck --error-exitcode=1 --quiet *.h *.c tests/ ; fi
+  - if [ -n "$CHECK" ]; then
+      make distcheck;
+      result=$?;
+      if [ $result != 0 ]; then

```

Comment:
No idea why this isn't working. I've used it successfully on another repo....

Reply:
> No idea why this isn't working. I've used it successfully on another repo....

Oh, my mistake; it was only displaying the logs because I had the condition wrong. Doesn't work. removed.